public/video-ui/src/util/validateField.js

```diff
@@ -0,0 +1,23 @@
+import FieldNotification from '../constants/FieldNotification';
+
+const validateField = (fieldValue, isRequired: false, isDesired: false) => {
```

Comment:
Yep, we pass validation functions to the specific field as an array and this func loops through them.